subsys/usb/usb_device.c

```diff
@@ -332,6 +332,11 @@ static void usb_handle_control_transfer(u8_t ep,
 
 		/* Send smallest of requested and offered length */
 		usb_dev.data_buf_residue = MIN(usb_dev.data_buf_len, length);
+		if (usb_dev.data_buf_len > CONFIG_USB_REQUEST_BUFFER_SIZE) {
+			LOG_ERR("REQUEST_BUFFER too small");
```

Comment:
I the same function above
https://github.com/zephyrproject-rtos/zephyr/blob/e6dc03550cd934e18a6748f437483be82f14c4b9/subsys/usb/usb_device.c#L306
there is similar logging based on wrong buffer length. It's done based on 'Data transfer direction' but I am wondering if its possible to concatenate this into one LOG_ERR to make it more clear? 

What is more. Is it really an error? Refering to 9.3.5 of usb 2.0 spec:

> On an input request, a device must never return more data than is indicated by the wLength value; it may
return less. On an output request, wLength will always indicate the exact amount of data to be sent by the
host. Device behavior is undefined if the host should send more data than is specified in wLength.